contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/metrics/data_profiler_metrics/data_profiler_table_column_list.py

```diff
@@ -37,7 +39,14 @@ def _pandas(
             column_names=profile_report_column_names,
             batch_columns_list=metrics["table.columns"],
         )
-        return profile_report_column_names
+        profile_report_filtered_column_names: list = []
```

Comment:
@alexjeilert If we are filtering in this different way, then do we still need the use of GX metrics for this purpose (the `get_dbms_compatible_column_names()`) call?  Also, the filtering code can be shortened using a list comprehension.  Thank you.